projects/pastanaga-angular/src/lib/date-picker/date-picker.component.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../styles/variables';
+
+:host {
+    --day-width: 32px;
+    --header-color: #{$color-neutral-strong};
+    --text-color: #{$color-dark-stronger};
+    --text-color-other-month: #{$color-neutral-light};
+    --weekday-border-color: #{$color-neutral-light};
+    --selected-background-color: #{$color-primary-lightest};
+    --list-item-background-color-hover: #{$color-neutral-lighter};
+
+    .calendar-popup {
+        font-size: 14px;
+        line-height: 24px;
+        color: var(--text-color);
+
+        ::ng-deep {
+            .pa-popup {
+                width: 320px;
+            }
+        }
+    }
+
+    .calendar-header {
+        display: flex;
+        align-items: center;
+
+        button {
+            border: none;
+            background: none;
+            outline: 0;
+
+            &:focus {
+                box-shadow: $shadow-focus !important;
+            }
+        }
+
+        .spacer {
+            flex-grow: 1;
+        }
+    }
+
+    .calendar-weekdays,
+    .calendar-row {
+        display: flex;
+        color: var(--header-color);
+        padding: 6px 0;
```

Comment:
6px is not part of our rhythm map, so there is a big chance that this value is wrong, or meant to be coming from some alignment flex rule more than a static padding.